File: src/main/java/akostenko/aicars/model/EnvironmentModel.java
package akostenko.aicars.model;

public interface EnvironmentModel {
    /** <i>m/s^2</i> */
    double g = 9.81;
    /** <i>kg / m^3</i> */
    double airDensity = 1.204;
    int SECONDS_PER_MINUTE = 60;
}

File: src/main/java/akostenko/aicars/model/EnvironmentModel.kt
package akostenko.aicars.model

interface EnvironmentModel {
    companion object {
        /** *m/s^2*  */
        val g = 9.81
        /** *kg / m^3*  */
        val airDensity = 1.204
        val SECONDS_PER_MINUTE = 60
    }
}
